src/DIRAC/Resources/Computing/SingularityComputingElement.py

```diff
@@ -449,6 +445,9 @@ def submitJob(self, executableFile, proxy=None, **kwargs):
             containerOpts = self.ceParameters["ContainerOptions"].split(",")
             for opt in containerOpts:
                 cmd.extend([opt.strip()])
+
+        rootImage = self.ceParameters.get("ContainerRoot") or findImage() or CONTAINER_DEFROOT
```

Comment:
```suggestion
        rootImage = findImage() or self.ceParameters.get("ContainerRoot") or  CONTAINER_DEFROOT
```

I think this would give me the possibility for VO-dependent configuration via the Pilot options?

Reply:
Correct!